src/neptune/metadata_containers/metadata_container.py

```diff
@@ -308,13 +308,12 @@ def _startup(self, debug_mode):
         self.start()
 
         if not debug_mode and self._mode not in {Mode.READ_ONLY}:
-            if in_interactive() or in_notebook():
+            if (in_interactive() or in_notebook()) and self.container_type == ContainerType.RUN:
                 logger.info(
-                    "Remember to stop your %s once you’ve finished logging your metadata (%s)."
-                    " It will be stopped automatically only when the notebook"
-                    " kernel/interactive console is terminated.",
+                    "You're in an interactive environment, to reduce monitoring time consumption some"
```

Comment:
@normandy7, there is nothing about such a warning in the specification so it's up to you if it should even exist :wink:

Reply:
I do think it could be useful to show :) Let me just tweak it a bit. I think we're shifting to the term "logging hours" instead of "monitoring time" so let's use that here.

Reply:
Ohh, thanks!